kiwi_keg/utils.py

```diff
@@ -53,7 +56,8 @@ def _get_source_files(root, src_path, ext, include_paths=None):
     return src_files
 
 
-def rmerge(src, dest):
+def rmerge(src: Dict[str, str], dest: Dict[str, str]) -> Dict[str, str]:
+    """Merge two dictionaries."""
```

Comment:
can we stick to the comment style which causes no problems to create api docs

```python
"""
Merge two dictionaries
"""
```

And the merge of two dicts is a one liner in python (dict.update) Thus if the comment is correct the extra function would be not needed and otherwise (which is the case) the comment should provide a little bit more context :)
What do you think ?